cluster/router/chain/chain.go

```diff
@@ -186,9 +186,6 @@ func (c *RouterChain) copyInvokerIfNecessary(cache *InvokerCache) []protocol.Inv
 func (c *RouterChain) buildCache() {
 	origin := c.loadCache()
 	invokers := c.copyInvokerIfNecessary(origin)
-	if invokers == nil || len(invokers) == 0 {
-		return
-	}
```

Comment:
why delete this block？

Reply:
delete empty protection.  When the address is empty, the route should be refresh.